module/mutect2.nf

```diff
@@ -90,12 +89,28 @@ workflow mutect2 {
             run_MergeMutectStats_GATK.out.merged_stats,
             run_LearnReadOrientationModel_GATK.out.read_orientation_model,
             contamination_table.collect()
+            )
+        filter_VCF_BCFtools(run_FilterMutectCalls_GATK.out.filtered
+            .map{ it -> ['all', it] }
+            )
+        split_VCF_BCFtools(filter_VCF_BCFtools.out.gzvcf
+            .map{ it -> it[1] },
+            ['snps', 'mnps', 'indels']
         )
-        filter_VCF_BCFtools(run_FilterMutectCalls_GATK.out.filtered.map{ it -> ['all', it] })
-        split_VCF_BCFtools(filter_VCF_BCFtools.out.gzvcf.map{ it -> it[1] }, ['snps', 'mnps', 'indels'])
-        compress_index_VCF(split_VCF_BCFtools.out.gzvcf)
-        file_for_sha512 = compress_index_VCF.out.index_out.map{ it -> ["mutect2-${it[0]}-vcf", it[1]] }
-            .mix( compress_index_VCF.out.index_out.map{ it -> ["mutect2-${it[0]}-index", it[2]] } )
+        rename_samples_BCFtools(
+            id_ch
+                .collect()
+                .combine(split_VCF_BCFtools.out.gzvcf)
+                .map { it.take(it.size() -2) }
```

Comment:
Consider adding some comment to clarify the numbers being used for maintenance/easier understanding